### ISSUE TITLE
New `"user"` field in `"info"` section of query jobs in `rethinkdb.jobs`

### ISSUE THREAD

**Issue by VeXocide** [rethinkdb/docs#1078]
The `rethinkdb.jobs` table now contains the username of the user running the query in a `"user"` field in the `"info"` section of the query job.

**Comment by chipotle** [comment]
Is that something that wouldn't show up if you ran from the data explorer? I just executed `r.db('rethinkdb').table('jobs')` and got back:

```
{
    "duration_sec": 0.000088,
    "id": [
        "query",
        "f3ab7b84-7dc0-498a-868f-b2dc3d13b31e"
    ],
    "info": {
        "client_address": "::1",
        "client_port": 51109,
        "query": "r.db(\"rethinkdb\").table(\"jobs\")"
    },
    "servers": [
        "watts_local_p91"
    ],
    "type": "query"
}
```

There's no `user` field in `info`, although there *is* a `query` field which isn't documented.

**Comment by danielmewes** [comment]
@chipotle The new `user` field isn't in 2.3 yet. This is only implemented in `next`, so it will be in `2.4.0`.
